scripts/_draw_info_display.py

```diff
@@ -0,0 +1,226 @@
+        text += f"    EV XYZ: {round(ev_xyz, digits)}\n\n"
+
+    if moving_road_velocity:
+        mrv_x = classes.VehiclePhysics.moving_road_velocity().x
+        mrv_y = classes.VehiclePhysics.moving_road_velocity().y
+        mrv_z = classes.VehiclePhysics.moving_road_velocity().z
+        text += f"     MRV X: {round(mrv_x, digits)}\n"
+        text += f"     MRV Y: {round(mrv_y, digits)}\n"
+        text += f"     MRV Z: {round(mrv_z, digits)}\n\n"
+    
+    if moving_road_velocity_xyz:
+        mrv_xyz = math.sqrt(classes.VehiclePhysics.moving_road_velocity().x ** 2
+                + classes.VehiclePhysics.moving_road_velocity().y ** 2
+                + classes.VehiclePhysics.moving_road_velocity().z ** 2)
+        text += f"   MRV XYZ: {round(mrv_xyz, digits)}\n\n"
+
+    if moving_water_velocity:
+        mwv_x = classes.VehiclePhysics.moving_water_velocity().x
+        mwv_y = classes.VehiclePhysics.moving_water_velocity().y
+        mwv_z = classes.VehiclePhysics.moving_water_velocity().z
+        text += f"     MWV X: {round(mwv_x, digits)}\n"
+        text += f"     MWV Y: {round(mwv_y, digits)}\n"
+        text += f"     MWV Z: {round(mwv_z, digits)}\n\n"
+
+    if moving_water_velocity_xyz:
+        mwv_xyz = math.sqrt(classes.VehiclePhysics.moving_water_velocity().x ** 2
+                + classes.VehiclePhysics.moving_water_velocity().y ** 2
+                + classes.VehiclePhysics.moving_water_velocity().z ** 2)
+        text += f"   MWV XYZ: {round(mwv_xyz, digits)}\n\n"
+
+    if charges_and_boosts:
+        mt = classes.KartMove.mt_charge()
+        smt = classes.KartMove.smt_charge()
+        ssmt = classes.KartMove.ssmt_charge()
+        mt_boost = classes.KartMove.mt_boost_timer()
+        trick_boost = classes.KartMove.trick_timer()
+        shroom_boost = classes.KartMove.mushroom_timer()
+        if classes.KartParam.is_bike() == 0:
+            text += f"MT Charge: {mt} ({smt}) | SSMT Charge: {ssmt}\n"
+        else:
+            text += f"MT Charge: {mt} | SSMT Charge: {smt}\n"
+        text += f"MT: {mt_boost} | Trick: {trick_boost} | Mushroom: {shroom_boost}\n\n"
+
+    if checkpoints_and_completion:
+        lap_comp = classes.RaceInfoPlayer.lap_completion()
+        race_comp = classes.RaceInfoPlayer.race_completion()
+        cp = classes.RaceInfoPlayer.checkpoint_id()
+        kcp = classes.RaceInfoPlayer.max_kcp()
+        rp = classes.RaceInfoPlayer.respawn_point()
+        text += f" Lap%: {round(lap_comp, digits)}\n"
+        text += f"Race%: {round(race_comp, digits)}\n"
+        text += f"CP: {cp} | KCP: {kcp} | RP: {rp}\n\n"
+
+    if miscellaneous:
+        wheelie_frames = classes.KartMove.wheelie_frames()
+        wheelie_cd = classes.KartMove.wheelie_cooldown()
+        trick_cd = classes.KartJump.cooldown()
+        airtime = classes.KartMove.airtime()
+        oob_timer = classes.KartCollide.solid_oob_timer()
+        if classes.KartParam.is_bike() == 1:
```

Comment:
Just personal preference, but I would probably write this is `if bool(classes.KartParam.is_bike()):`